k8s.gcr.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -1,11 +1,13 @@
 - name: cluster-api-controller
   dmap:
+    "sha256:9652a07cc57081be625d50ab68d4226c7db0e2289384dc1a7061b6a7375f0f05": ["v0.1.10"]
```

Comment:
@sbueringer @fabriziopandini note that the diff is bigger than expected because kpromo pr was changed to sort by tag lexicographically _not_ by semver tag, since it supports tags that are not semver too.

Reply:
ref https://github.com/kubernetes-sigs/promo-tools/pull/527